import/popups/AuthDialog.qml

```diff
@@ -38,6 +40,45 @@ Dialog {
     property alias preventDialogsPrefillValue: auth.preventDialogsPrefillValue
     property alias preventDialogsValue: auth.preventDialogsValue
 
+    // Trim obnoxiously long realms
+    property string trimmedRealm: realm.length <= 150 ? realm : realm.substring(0, 150) + "\u2026"
```

Comment:
Hmm.  The "obnoxious" length might depend on device type, orientation, etc.  I think using a magic value here might not make the most sense, but instead maybe a constant from dconf would be better?  It's not a huge deal, obviously, but otherwise this is one of those "variation points" which isn't obvious.

Reply:
I stole your comment from #115 to make things clearer. I agree with your idea to have some kind of configurable value here, but maybe it's something we could look at separately, using a reusable utility function (going by the comment in the gecko code, we should probably be localising the ellipsis, but then things start to get more complicated).

Reply:
Now that the key ids and parameters are passed as a bundle to the UI, the trimming now happens in gecko again rather than the QML, so this bit of code has been removed.